### ISSUE TITLE
Does anyone have successfully used Borsh::serialize()?

### ISSUE THREAD

**Issue by mindell** [tighten/solana-php-sdk#35]
When i tried Borsh::serialize() i got this error..
```php
Call to undefined method Tighten\\SolanaPhpSdk\\Borsh\\BinaryWriter::writeTighten\\SolanaPhpSdk\\Accounts\\Creator()
```

**Comment by neverything** [comment]
According to the tests is seems to be working: https://github.com/tighten/solana-php-sdk/blob/main/tests/Unit/BorshTest.php

**Comment by mindell** [comment]
Yes, but deserializing NFT and then serializing it again is not working. 
Anyway, I already use ts-node on that part.

**Comment by neverything** [comment]
@mindell this does indeed seem like a bug currently. Feel free to write a pull request for it or I'll get to it eventually in the future.
